src/SmartComponents/Recs/List.js

```diff
@@ -100,7 +100,7 @@ const List = () => {
                 }
               >
                 {activeTab === PATHWAYS_TAB && (
-                  <Suspense fallback={<Loading />}>
+                  <Suspense fallback={<SkeletonTable />}>
```

Comment:
Not sure about this, maybe a spinner is better since this SkeletonTable is propless and `PathwaysTable` component has a SkeletonTable inside of itself.